CHANGELOG.md

```diff
@@ -18,6 +20,214 @@
+- `kong.db.errors` receives the name of the current strategy [#3612](https://github.com/Kong/kong/pull/3612)
+- The new DAO has the correct CRUD event [#3659](https://github.com/Kong/kong/pull/3659)
+  and triggers update events only once [#4095](https://github.com/Kong/kong/pull/4095)
+- SNIs are correctly paginated [#3722](https://github.com/Kong/kong/pull/3722)
+- null & default values are handled better
+  [#3772](https://github.com/Kong/kong/pull/3772),
+  [#3710](https://github.com/Kong/kong/pull/3710),
+  [#3910](https://github.com/Kong/kong/pull/3910)
+- Inferring arguments with `application/x-www-form-urlencoded` on the admin API
+  works better [#3770](https://github.com/Kong/kong/pull/3770)
+- The cache_key field is properly calculated [#3831](https://github.com/Kong/kong/pulls/3831)
+- `$request_uri` being nil now is handled correctly [#3842](https://github.com/Kong/kong/pull/3842)
+- Primary key violations in Cassandra are detected more accurately [#3865](https://github.com/Kong/kong/pull/3865)
+- Datastax Enterprise 6.X doesn't throw errors (thanks, [@gchristidis](https://github.com/gchristidis)) [#3873](https://github.com/Kong/kong/pull/3873)
+- Posgresql's `connector.iterate` function is used properly [#3887](https://github.com/Kong/kong/pull/3887)
+- Conditionals can handle structured data correctly in the schema [#3936](https://github.com/Kong/kong/pull/3936)
+- Recently-created upstreams return `[]` instead of `{}` for their initial
+  list of targets [#4058](https://github.com/Kong/kong/pull/4058)
+- The postgres strategy creates array index expressions correctly [#4078](https://github.com/Kong/kong/pull/4078)
+- The semaphore protecting the router is always released [#4084](https://github.com/Kong/kong/pull/4084) & [#4098](https://github.com/Kong/kong/pull/4098)
+- The router no longer injects an extra `/` in certain cases [#3780](https://github.com/kong/kong/pull/3780)
+- Several typos, style and grammar fixes by [@saideepd](https://github.com/saideepd), [@gy741](https://github.com/gy741), [@arpitpandey0209](https://github.com/arpitpandey0209), [@joelvisroman](https://github.com/joealvisroman), [@vkmrishad](https://github.com/vkmrishad), [@mr-yamraj](https://github.com/mr-yamraj), [@geekysrm](https://github.com/geekysrm), [@Mehvix](https://github.com/Mehvix), [@vyaspranjal33](https://github.com/vyaspranjal33), [@iTechTR](https://github.com/iTechTR), [@shyamjalan ](https://github.com/shyamjalan ), and [@steffinstanly](https://github.com/@steffinstanly).
+
+
+##### Plugins
+
+- Record fields are not nullable in plugin schemas [#3778](https://github.com/Kong/kong/pulls/3778)
+- Fixed an issue where some plugins could include default HTML responses from
+  Lapis [#4077](https://github.com/Kong/kong/pull/4077)
+- cors:
+  - set 'Vary: Origin' when ACAC is enabled (thanks, (@marckhouzam)[https://github.com/marckhouzam]!) [#3675](https://github.com/Kong/kong/pull/3765)
```

Comment:
Needed to google what `ACAC` is.

Reply:
Good call here. I also dislike random initials. Replaced with Access-Control-Allow-Credentials.